tests/testlib/s2n_connection_test_utils.c

```diff
@@ -114,16 +100,25 @@ void s2n_print_connection(struct s2n_connection *conn, const char *marker)
         }
     }
     printf("\n");
+}
 
-    printf("OUT Stuffer (write: %d, read: %d, size: %d)\n", conn->out.write_cursor, conn->out.read_cursor, conn->out.blob.size);
-    for (i = 0; i < conn->out.write_cursor; i++) {
-        printf("%02x", conn->out.blob.data[i]);
-        if ((i + 1) % 8 == 0) {
-            printf(" ");
-        }
-        if ((i + 1) % 40 == 0) {
-            printf("\n");
-        }
-    }
-    printf("\n");
+void s2n_print_blob(struct s2n_blob *blob, const char *name)
+{
+    printf("%s Blob (size: %d)\n", name, blob->size);
+    s2n_print_bytearray(blob->data, blob->size);
+}
+
+void s2n_print_stuffer(struct s2n_stuffer *stuffer, const char *name)
```

Comment:
`printf()` can fail with a negative error code.  http://www.cplusplus.com/reference/cstdio/printf/ 
It might make sense to have these functions also able to return an error code if that happens.